### ISSUE TITLE
Integration will stop working with home assistant 2021.6

### ISSUE THREAD

**Issue by nckmml** [nstrelow/ha_philips_android_tv#56]
From logs:

No 'version' key in the manifest file for custom integration 'philips_android_tv'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'philips_android_tv'

**Comment by atozfugh** [comment]
Yeah, i got the same problem. If i can help somehow (provide logs etc.) feel free to message me!
